terraform/main.tf

```diff
@@ -1,3 +1,12 @@
+data "terraform_remote_state" "domain_prod_gcp_infra" {
+  backend   = "gcs"
+  workspace = terraform.workspace
+  config = {
+    bucket = "im-common-cicd-infra"
+    prefix = "terraform/state/im-data-gcp-department-infra"
+  }
+}
+
```

Comment:
I don't see any change relating to the above block being added, what is it for?

Reply:
Good pickup. Not sure how it came in there. Removed.